src/main/java/com/mraof/minestuck/item/block/MultiblockItem.java

```diff
@@ -51,24 +54,36 @@ public InteractionResult place(BlockPlaceContext context)
 	{
 		Level level = context.getLevel();
 		Direction sideFace = context.getClickedFace();
-		if (level.isClientSide)
-		{
-			return InteractionResult.SUCCESS;
-		} else if (sideFace != Direction.UP)
+		if (sideFace != Direction.UP)
 		{
 			return InteractionResult.FAIL;
 		} else
 		{
 			Direction facing = context.getHorizontalDirection().getOpposite();
 			BlockPos pos = getPlacementPos(context);
+			BlockPos clickedPos = context.getClickedPos();
+			Player player = context.getPlayer();
+			ItemStack stack = context.getItemInHand();
 			
 			if(!canPlaceAt(context, pos, facing))
 				return InteractionResult.FAIL;
 			
-			BlockState state = getBlock().defaultBlockState().setValue(AlchemiterBlock.FACING, facing);
+			BlockState state = getBlock().defaultBlockState();
 			this.placeBlock(context, state);
-			updateCustomBlockEntityTag(pos, level, context.getPlayer(), context.getItemInHand(), state);
-			return InteractionResult.SUCCESS;
+			updateCustomBlockEntityTag(pos, level, player, stack, state);
+			
+			if (player instanceof ServerPlayer serverPlayer) {
+				CriteriaTriggers.PLACED_BLOCK.trigger(serverPlayer, clickedPos, stack);
+			}
+			
+			level.gameEvent(player, GameEvent.BLOCK_PLACE, clickedPos);
+			SoundType soundtype = state.getSoundType(level, clickedPos, context.getPlayer());
+			level.playSound(player, clickedPos, this.getPlaceSound(state, level, clickedPos, player), SoundSource.BLOCKS, (soundtype.getVolume() + 1) / 2, soundtype.getPitch() * 0.8F);
+			if (player == null || !player.getAbilities().instabuild) {
+				stack.shrink(1);
+			}
+			
+			return InteractionResult.sidedSuccess(level.isClientSide);
```

Comment:
You used an inappropriate method when merging mraof:1.18, which copied bae517355f43b4fd978551119bfac19746e9bdbf and 9c4d3b61c16481ad70c8ec4a85847309553f9722 as new commits ba19ed26f199ba00d28ce5727def2163cba5dba5 and 2feb29898ed51e677fe905ab50a3c57b78da1b7b. While nothing need to be done about it this time, make sure next time to perform a regular merge (or possibly a rebase as long as you understand the consequences of it).

Reply:
I thought I did a regular merge, but I might have made a mistake when using intelliJ's built-in git features.